components/steplist/index.css

```diff
@@ -61,6 +34,16 @@ governing permissions and limitations under the License.
     /* Show cursor on hit zone when clickable */
     cursor: pointer;
   }
+  .spectrum-Steplist-item {
+    /* Step Focused */
+    &:focus,
+    &.is-focused,
+    *:focus {
+      .spectrum-Steplist-marker {
+        background-color: var(--spectrum-blue-700);
```

Comment:
Rather than referencing these global variables directly, we like to create our own variable name for the component, similar to what was happening in the file originally. Up at the beginning of the file you could create the variable 
```
--spectrum-steplist-market-background-color: var(--spectrum-blue-700);
```
And then reference the specific steplist variable here.

Reply:
This also does not seem to be the correct blue.. For `spectrum--light` the current blue used is rgb(20, 122, 243) but it looks like this resolves to rgb(56, 146, 243)